csv_importer/csv_importer/models.py

```diff
@@ -36,6 +62,16 @@
     Column('place_id', ForeignKey('place.id'), nullable=False)
 )
 
+# Examination table representation one examination session or "Изпитна сесия".
```

Comment:
For DZIs, there are two sessions per year - May/June and August/September. The government portal currently includes only the May/June but the other data is available on request. So this is something we may want to include eventually noting that the data import will be different. So, I am wondering whether this calls for the inclusion of another column  e.g. `month` in the examination table. Probably worth discussing this in more detail offline.

Reply:
I agree we should discuss this.
I like the idea for `month` column.

I guess this additional autumn session will have fewer number of students attending, is that true? Are those students such who skipped the summer session or such that want to get better score?